controller/tap/server.go

```diff
@@ -68,7 +68,7 @@ func (s *server) TapByResource(req *public.TapByResourceRequest, stream pb.Tap_T
 		}
 
 		for _, pod := range podsFor {
-			if pkgK8s.IsMeshed(pod, s.controllerNamespace) {
+			if pkgK8s.IsMeshed(pod, s.controllerNamespace) && !pkgK8s.IsTapDisabled(pod) {
```

Comment:
tioli, the `if len(pods) == 0` below returns a pretty generic `no pods found for...` message. consider returning a more descriptive message if pods were filtered out due to tap disabling.